engine/src/test/java/org/hibernate/search/test/backend/lucene/ResourcesClosedInOrderTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.test.backend.lucene;
+
+import org.hibernate.search.backend.spi.Work;
+import org.hibernate.search.backend.spi.WorkType;
+import org.hibernate.search.spi.SearchIntegrator;
+import org.hibernate.search.spi.SearchIntegratorBuilder;
+import org.hibernate.search.testsupport.BytemanHelperStateCleanup;
+import org.hibernate.search.testsupport.setup.CountingErrorHandler;
+import org.hibernate.search.testsupport.setup.SearchConfigurationForTest;
+import org.hibernate.search.testsupport.setup.TransactionContextForTest;
+import org.jboss.byteman.contrib.bmunit.BMRule;
+import org.jboss.byteman.contrib.bmunit.BMRules;
+import org.jboss.byteman.contrib.bmunit.BMUnitRunner;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.hibernate.search.testsupport.BytemanHelper.isEventStackEmpty;
```

Comment:
I'm fine either way, I think I always had them last (long before Eclipse did mess with them) so the habit (and somewhat my expectation) has them on the bottom but we can change that if you prefer.